sys/include/net/gnrc/gnrc_netdev2.h

```diff
@@ -9,14 +9,14 @@
 /**
  * @defgroup  net_gnrc_netdev2   Adaption layer for GNRC on top of Netdev2
  * @ingroup   net_gnrc
- * @brief     Provides the glue code for @ref gnrc on top of @ref drivers_netdev_netdev2
+ * @brief     Provides the glue code for @ref net_gnrc on top of @ref drivers_netdev_netdev2
  * @{
  *
  * @file
- * @brief     netdev2 gnrc glue code interface
+ * @brief     netdev2-GNRC glue code interface
```

Comment:
From my standpoint, yes. But in general there are mixed occurrences in this file, so I unified it to the form more often used.